### DISCUSSION TITLE
Default config?

### DISCUSSION THREAD

**Discussion by SiddharthShyniben** [w0rp/ale#3954] [Ideas]
It would be nice to be able to provide default config for all LSPs in central location like in a config.json or a folder in the home directory instead of having to create, say, a `.jshintrc` with `esversion: 6` in every project I have.

I have no idea how this could be achieved, just an open thought.

**Comment by w0rp** [comment]
I never want to add configuration file support to ALE for two reasons.

1. Each tool should be configured with configuration files specific to the tools so configuration is shared between developers, including those who don't use Vim.
2. For Vim, there are existing plugins for loading configuration files that can configure not only ALE, but any Vim plugin.
